lib/CL/devices/remote/communication.c

```diff
@@ -2362,13 +2362,26 @@ pocl_network_setup_peer_mesh ()
  *
  */
 cl_int
-pocl_network_build_or_link_program (
-    remote_device_data_t *ddata, const void *payload, size_t payload_size,
-    int is_binary, int is_builtin, int is_spirv, uint32_t prog_id,
-    const char *options, char **kernel_meta_bytes, size_t *kernel_meta_size,
-    uint32_t *devices, uint32_t *platforms, size_t num_devices,
-    char **build_logs, char **binaries, size_t *binary_sizes,
-    size_t svm_region_offset, int compile_only, int link_only)
+pocl_network_build_or_link_program (remote_device_data_t *ddata,
```

Comment:
Is this really the format clang-format prefers here for the long argument list?

Reply:
I ran `format-last-commit.sh` again just to be sure and that what it did.